src/components/Categories.js

```diff
@@ -1,10 +1,20 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
 
-const Category = () => (
-  <section>
-    <div>
-      <button type="submit">Check status</button>
-    </div>
-  </section>
-);
+const Category = () => {
+  const dispatch = useDispatch();
+
+  const handleStatusChange = () => {
+    dispatch(checkStatus());
+  };
+
+  return (
+    <section>
+      <div>
+        <button type="submit" onClick={handleStatusChange}>Check status</button>
+      </div>
+    </section>
+  );
+};
```

Comment:
This is a requirement but it is optional at this point because it was missed previously.

- [ ] **[OPTIONAL]** When I click the "check status" button on the categories page, nothing happens. After dispatching the action to the store, you need to fetch the category state and display "Under construction" in the categories page as stated in the requirements. Kindly do that.